
public class TestException3 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		try {
			throw new RuntimeException();
			
		}catch(Exception e) {
			System.out.println("RunRimeException");
		}
	}
		

	

}

